src/components/atoms/ChatBubble/ChatBubble.js

```diff
@@ -97,27 +120,27 @@ const ChatBubble = props => {
   });
 
   return (
-    <Bubble alignment={alignment} colorTheme={colorTheme} style={style} z={z}>
+    <Bubble className={className} alignment={alignment} colorTheme={colorTheme} style={style} z={z}>
       <ContentWrapper>
-        <div>
+        <BubbleContent>
           {children ||
             (content ? <BubbleText colorTheme={colorTheme}>{content}</BubbleText> : null)}
-        </div>
-        {/*
-          TODO: Implement Atom Icon Component
-          {iconRight && onClickIconRight ? (
+        </BubbleContent>
+
+        {iconRight && onClickIconRight ? (
           <Aside>
-            <IconButton onClick={onClickIconRight} z={0}>
+            <IconButton onClick={onClickIconRight} color="white" z={0}>
               <Icon color={theme.chatBubble[colorTheme].asideIcon} name={iconRight} />
```

Comment:
ESLint: 'theme.chatBubble' and .asideIcon missing in props validation(react/prop-types)